druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/namespace/RocksDBExtractionNamespace.java

```diff
@@ -13,8 +13,8 @@
 import javax.validation.constraints.NotNull;
 import java.util.Objects;
 
-@JsonTypeName("mahainmemorydb")
```

Comment:
this rename is backward incompatible change.

Reply:
I believe this is only used for serialization.